### DISCUSSION TITLE
Wii U Pro controller connects, but isn't controlling anything

### DISCUSSION THREAD

**Discussion by podings** [KeyPuncher/WiinUPro#170] [Help]
![WiinUPro2](https://github.com/KeyPuncher/WiinUPro/assets/8339617/e383a5c8-2813-441e-ae66-85b093fc6453)
![WiinUPro1](https://github.com/KeyPuncher/WiinUPro/assets/8339617/7b5e17e1-af68-4f9a-8ba6-14b3810bc7a2)

I can get my Wii U Pro controller to connect, and the input is recognised by WiinUPro.
But it simply doesn't translate into actually controlling anything in Windows.

Any Xbox controller I connect seems to work just fine, so I tak it isn't to do with any of those drivers missing.

![image](https://github.com/KeyPuncher/WiinUPro/assets/8339617/60d31811-2a1a-407c-9312-967cda8326d2)

Same thing in WiinUSoft
The controller identifies with a rumble when asked to, and can be set to identify as Xbox Controller 1.
But it doesn't actually do anything in any games.

UPDATE:

After a bit of mucking about, WiinUSoft seems to be working as intended, mapping both input and rumble output correctly for games!

The only thing that appears to NOT work is the automapping of D-pad to arrow key input for HTML5 games and similar.
Which, I might add, appears to now be a consistent problem on my Windows 10 installation for all controller input, Xbox controllers as well.

So feel free to entirely disregard that issue, or indeed this whole topic as a whole, unless you have any specific questions.

**Comment by KeyPuncher** [comment]
Glad it's working now, just with WiinUPro make sure to load a profile to get activate those mappings.
